tests/lib.rs

```diff
@@ -85,6 +87,7 @@ fn arenatree_iter() {
     assert_eq!(node_refs, vec![&arena[a], &arena[b], &arena[c], &arena[d]]);
 }
 
+#[cfg(feature = "par_iter")]
```

Comment:
Can you please change the appveyor and travis config to also test with this feature?